source/MRViewer/MRPlaneVisualizer.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+#include "MRMesh/MRBox.h"
+#include "MRMesh/MRPlane3.h"
+#include "MRViewer/MRViewer.h"
+
+namespace MR
+{
+struct PlaneVisualizer
```

Comment:
mb name it PlaneWidget?

Reply:
I think this widget must be more simple, it should own Plane3f and mb `last transform`,
`bool planeIsDefined ~ bool(planeObj)`
`bool showPlane ~ planeObj->isVisibile()`
other should be controlled by plugins and `ImGui::Plane`

mb it also should have frame border properties (border should be optional) and methods like
`
update( Plane, Box ,bool updateCameraRotation = true) // +setup(if is not defined)
reset() // ~undefine
`

it should subscribe to mouse events itself, like ObjectTransformWidget